graphics/vtk/Portfile

```diff
@@ -117,7 +115,7 @@ variant qt5 description {Add Qt5 support.} {
 }
 
 # Supported pythons
-set python_versions {27 36 37 38}
+set python_versions {27 36 37 38 39}
```

Comment:
you cannot add Python 3.9 here as the `py39-mpi4py` subport (used below `if {[mpi_variant_isset]} {`) is not present

Reply:
Does anyone know what the plan is, relative to creating python 3.9 subports for such modules? There are a number of cases where they're needed, but don't yet exist.

Is this simply a matter of folks submitting pull requests with the additions, when needed?

@ryandesign @jmroot

Reply:
@mascguy well, if you want to add support for PY39 one need to make sure that all its dependencies are available. So in this PR one would need to add `py39-mpi4py`. Or the py39 subport is removed here for now and this PR can be merged. If someone wants to use Python 3.9 it will, of course, still require all dependencies to be added but that can happen in another PR.

Reply:
Makes sense Renee.